environments/dev-env/README.md

```diff
@@ -0,0 +1,394 @@
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+For MacOS only:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/README.md)
+
+
+## Quickstart
+
+#### 1. Clone this repo:
+
+```bash
+git clone -b dev-env-v3 https://github.com/dumb0002/edge-mc.git
```

Comment:
This literal URL will become wrong as soon as this PR merges.  That is bad.  I recommend writing something that does not become incorrect as soon as it merges.  Perhaps just saying "clone this repo" (along with adjusting later references to the local directory to make it clear that they refer to what `git clone` produced) is enough?  The reader knows what repo they are looking at, and we should assume that developers know how to use `git clone`.